### ISSUE TITLE
[WIP][P2][Fd-inline help][Sourcing][Accessibility] Role tooltip is not being read on accessing a tooltip in JAWS screenreader

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#11118]
#### Is this a bug, enhancement, or feature request?
bug

#### Describe your proposal.
The role tooltip should be read when focus is on "Hover on me to see inline help" element and a tooltips is displayed on screen.
This is noticed by ATL team

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

Angular: 15
core / platform: 0.43.22

#### Please provide relevant source code (if applicable).
Replicable on https://ng-15-downport--fundamental-ngx.netlify.app/#/core/inline-help#template

#### Please provide stackblitz example(s).
https://ng-15-downport--fundamental-ngx.netlify.app/#/core/inline-help#template

**Comment by sagar7259** [comment]
The role tooltip should be read for the users on accessing the icon, here reading only the text within the tooltip is not sufficient has the role or UI type "Tooltip/Hint" is not read.

**Comment by droshev** [comment]
@nikolay-kolarov Do you think you can advise us something here?

**Comment by nikolay-kolarov** [comment]
Hi @sagar7259,

Could you provide more info on why you expect additional tooltip announcement? Did you tested a more complex example where this is required?

In the native HTML tooltips there is no additional announcement for the basic text tooltips.
You can try with a standard HTML button and see that the title is simply read as part of the description `<button title="Additional info">Test</button>`.

Thanks,
Nikolay

**Comment by sagar7259** [comment]
Hello Nikolay,

I guess you have misunderstood the issue here. 

When this tooltip icon is accessed, the information within this icon is simply being read to the users. But screen reader users will not know that they are accessing a help tooltip.

Therefore the role for this icon button is very important to read to the users where they will understand that it is a help tooltip.

https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/

Regards,
Sagar

**Comment by nikolay-kolarov** [comment]
Hello @sagar7259,

The reference in APG still do not have an official example, but one can sneak in https://github.com/w3c/aria-practices/issues/128 and e.g. see this:
`<button ... aria-describedby="p1">...</button>
<div role="tooltip" id="p1"...>Lipsum...</div>`
So basically they refer an element with role tooltip via the aria-describedby.
And what I meant was that this will simply announce the text of the tooltip as description of the focused element and will not say explicitly that this is a tooltip. This is the same for the native HTML tooltips (set with the title attribute in HTML).

That is why I asked why you want to hear explicitly the tooltip role. Have you experienced somewhere such an announcement?

Thanks,
Nikolay

**Comment by sagar7259** [comment]
Hello Nikolay,

Thanks for your input here. Has discussed with our ACC experts we can close this ticket has there is no need of additional announcement for the basic text tooltips and we agree with your previous input.

@rvamudha we can close this.
 
Thanks and Regards,
Sagar

**Comment by I543348** [comment]
closing as per ATL comments
Thanks
